### PR TITLE
Ft account api

### PR DESCRIPTION
**Pull request by RevanthTanneeru**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [ x] You created a branch from `develop` branch.
- [ x] You PR is raised on `develop` branch and not on`main`.
- [ x] You have read the [Contribution Guidlines](CONTRIBUTING.md) before creating this PR.

### Description of the Change

I have added add schema command except for the advanced configuration part. Changed formats of get and delete schema commands and refracted code.

### CODE REVIEW

File: cmd/add/schema/schema.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	err2 = json.Unmarshal(resp1, &resultResp1)
+	if err2 != nil {
+		return err2
+	}
+	resultResp.Data[temp1[temp-1]].Enabled = true
+	var DisplayName string
+	var req string
+	fmt.Print("Enter the Display Name (" + resultResp.Data[temp1[temp-1]].Display + ") :")
+	scanner := bufio.NewScanner(os.Stdin)
+	scanner.Scan()
```
Comment (mohammed786) [inline cmd/add/schema/schema.go:115]: Can't we use `fmt.Scanln` function here?

File: cmd/add/schema/schema.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	fmt.Print("Is Required (y/n):")
+	fmt.Scanln(&req)
+	if req == "Y" || req == "y" {
+		resultResp.Data[temp1[temp-1]].IsMandatory = true
+	} else if req == "N" || req == "n" {
+		resultResp.Data[temp1[temp-1]].IsMandatory = false
+	} else {
+		fmt.Println("please type in (y/n)")
+	}
+	fmt.Println("Do you want to set Advance Configuiration for this feild(y/n):")
```
Comment (mohammed786) [inline cmd/add/schema/schema.go:126]: Do we need this line as we are not doing advance configuration for now right?

**Review by mohammed786 — CHANGES_REQUESTED** [review]
Please have a look at the comment once, and also have you tested weather the field is showing up in `idx` page after adding it through cli?

File: cmd/add/schema/schema.go (hunk 1 above)
Comment (RevanthTanneeru) [reply cmd/add/schema/schema.go:115]: If we use scanln we can’t read display names with spaces in them. So I’m using `bufio.NewScanner(os.Stdin)`

**Review by mohammed786 — APPROVED** [review]
Looks good

**Pull request merged**